xbmc/filesystem/IFileTypes.h

```diff
@@ -83,4 +84,15 @@ enum CURLOPTIONTYPE
   CURL_OPTION_HEADER      /**< Add a Header           */
 };
 
+enum FileProperty
+{
+  FILE_PROPERTY_RESPONSE_PROTOCOL,          /**< Get response protocol line  */
+  FILE_PROPERTY_RESPONSE_HEADER,            /**< Get response Header value  */
+  FILE_PROPERTY_CONTENT_TYPE,               /**< Get file content-type  */
+  FILE_PROPERTY_CONTENT_CHARSET,            /**< Get file content charset  */
+  FILE_PROPERTY_MIME_TYPE                   /**< Get file mime type  */
+};
+
+/* default content-type for use in returns by ref */
```

Comment:
Always a good idea to define constants, but I woul remove the part re return by ref. It is a good thing also if not returning refs. And actually your code uses the constant only in a scenario where it is returned by value. ;-)